ray-operator/main.go

```diff
@@ -167,6 +168,12 @@ func main() {
 	exitOnError(ray.NewRayJobReconciler(mgr).SetupWithManager(mgr),
 		"unable to create controller", "controller", "RayJob")
 
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
+		if err = (&rayv1.RayCluster{}).SetupWebhookWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create webhook", "webhook", "RayCluster")
+			os.Exit(1)
+		}
+	}
```

Comment:
copied from kubebuilder docs

- https://book.kubebuilder.io/cronjob-tutorial/running.html?highlight=ENABLE_WEBHOOKS#optional
- https://book.kubebuilder.io/cronjob-tutorial/main-revisited.html?highlight=ENABLE_WEBHOOKS#you-said-something-about-main

Reply:
@kevin85421 shall we invert this conditional so that users have to explicitly set the env var to enable the hooks?

Reply:
Yes, disabling it by default is better. Many users are not allowed to install webhooks.

Reply:
updated